### PR TITLE
Bookstore(Styling)

### PR DESCRIPTION
**Pull request by edubew**
## This milestone fulfills the following project requirements:
- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Books/books.css [hunk 1]
```diff
@@ -1,51 +1,28 @@
...
 }
 
-.book__container > small {
+.book__content > small {
   color: #999;
   font-weight: 800;
 }
 
-.book__container > h2 {
+.book__content > h2 {
```
Comment (franclobo) [inline src/components/Books/books.css:30]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab), align the text to the left at the same level as the rest, and adjust the container size and margins to match the Zeplin design 👇🏽 

image

- [x] There is a border solid around the panel, please 🙏🏽 add it to match the Zeplin design 👇🏽 

image


| Zeplin | Your design |
|---|---|
| image | image |

File: src/index.css [hunk 2]
```diff
@@ -25,3 +26,36 @@ code {
...
+  background: #fff;
+  box-shadow: 1px 2px 2px rgb(218, 214, 214);
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 2rem;
+  font-weight: 600;
+  color: rgb(51, 182, 226);
+}
```
Comment (franclobo) [inline src/index.css:46]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [Montserrat](https://fonts.google.com/specimen/Montserrat) to match the Zeplin design 👇🏽 

image

File: src/index.css [hunk 3]
```diff
@@ -25,3 +26,36 @@ code {
...
+  gap: 2rem;
+  margin-right: 50%;
+}
+
+.nav__item {
+  text-decoration: none;
+  font-size: 1.1rem;
+  right: 50%;
+  color: #000;
+}
```
Comment (franclobo) [inline src/index.css]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [Montserrat](https://fonts.google.com/specimen/Montserrat), align them, and use the correct contrast to match the Zeplin design.

| Zeplin | Your design |
|---|---|
| image | image |

File: src/components/Books/books.css [hunk 4]
```diff
@@ -1,51 +1,28 @@
...
   margin: 2px;
 }
 
-.book__container p {
+.book__content p {
```
Comment (franclobo) [inline src/components/Books/books.css:36]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab) to match the Zeplin design.

File: src/components/Books/Book.js [hunk 5]
```diff
@@ -12,13 +12,31 @@ const Book = ({
...
+        <small>Fictional</small>
+        <h2>{title}</h2>
+        <p>{author}</p>
+        <div className="cta__container">
+          <button type="submit">Comments</button>
+          <button type="submit" onClick={deleteBook}>
+            Remove
+          </button>
+          <button type="submit">Edit</button>
+        </div>
```
Comment (franclobo) [inline src/components/Books/Book.js:25]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab), and add a bar ( | ) between the links to match the Zeplin design. 👇🏽 

| Zeplin | Your design |
|---|---|
| image | image |

File: src/components/Books/books.css [hunk 6]
```diff
@@ -61,6 +38,62 @@ nav {
...
+
+.current__chapter {
+  color: #999;
+  margin-bottom: 0;
+}
+
+.update__container p {
+  margin-top: 2px;
+  font-size: 1.1rem;
+}
```
Comment (franclobo) [inline src/components/Books/books.css]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab) to match the Zeplin design.

File: src/components/Books/books.css [hunk 7]
```diff
@@ -61,6 +38,62 @@ nav {
...
+  right: 15%;
+  width: 2px;
+  height: 4rem;
+  background: rgb(223, 216, 216);
+}
+
+.current__chapter {
+  color: #999;
+  margin-bottom: 0;
+}
```
Comment (franclobo) [inline src/components/Books/books.css]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab) to match the Zeplin design

File: src/components/Books/books.css [hunk 8]
```diff
@@ -61,6 +38,62 @@ nav {
...
+
+.progress__btn {
+  background-color: rgb(51, 182, 226);
+  color: #fff;
+  font-weight: 600;
+  border: none;
+  padding: 0.8rem;
+  margin-top: 1.5rem;
+  border-radius: 0.2rem;
+}
```
Comment (franclobo) [inline src/components/Books/books.css]: - [ ] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [RobotoSlab](https://fonts.google.com/specimen/Roboto+Slab) to match the Zeplin design.

File: src/components/Books/books.css [hunk 9]
```diff
@@ -1,51 +1,28 @@
...
   list-style: none;
   margin-bottom: 1.2rem;
+  margin-left: 5%;
+  margin-right: 5%;
   padding: 1rem;
-  border: 1px solid rgb(235, 221, 221);
 }
 
-.book__container > small {
+.book__content > small {
```
Comment (franclobo) [inline src/components/Books/books.css:24]: - [x] Styling must be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/). Please add the font family [Montserrat](https://fonts.google.com/specimen/Montserrat) to match the Zeplin design.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @edubew,

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- Add form works as expected ✔️ 
- Remove button works as expected ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arslanbisharat** [comment]
Hi @edubew, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no changes after last review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by edubew** [comment]
Hello @arslanbisharat ,
I have implemented all the changes the previous reviewer asked for before re-submitting my PR for a re-review. I'm a little confused when you say that I haven't made any changes.
Here's a link to the last commit I made after @franclobo reviewed my project:
`https://github.com/edubew/bookstore/pull/5/commits/e43fa50bfd680204018d9b910318364a475bc82e`

**Review by alexander16108 — APPROVED** [review]
Hi @edubew , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


## Highlights

📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**